src/components/TransactionBlock/AssetBridgeTransactionBlock.tsx

```diff
@@ -203,7 +208,17 @@ const AssetBridgeTransactionBlock = ({
           toAddress: receiverAddress ?? undefined,
         });
         setAvailableRoutes(routes);
-        if (routes.length === 1) setSelectedRoute(mapRouteToOption(routes[0]));
+        let gasUsdValues: (number | undefined)[] = [];
+        let receiveAmount: number[] = [];
+
+        routes.forEach(({ gasCostUSD, fromAmountUSD }) => {
+          gasUsdValues.push(gasCostUSD ? +gasCostUSD : undefined);
+          receiveAmount.push(+fromAmountUSD);
+        });
+
+        const bestRouteIndex = getOfferItemIndexByBestOffer(gasUsdValues, receiveAmount);
```

Comment:
this part is very hard to read, please move everything under `getOfferItemIndexByBestOffer`, thanks

Reply:
Created a separate function for asset bridge as it uses `Route[]` over `ExchangeOffer[]`